src/ynEIGEN/EigenStrategyManager.sol

```diff
@@ -306,7 +305,15 @@ contract EigenStrategyManager is
                     asset,
                     strategies[asset].userUnderlyingView((address(node)))
                 );
-                stakedBalances[j] += strategyBalance;
+
+                uint256 strategyWithdrawalQueueBalance;
+                uint256 queuedShares = node.queuedShares(strategies[asset]);
+                if (queuedShares > 0)
```

Comment:
let's please use { } all the time, eliminates the possibility of making a block scope mistkae.